master/reference/repo-structure.md

```diff
@@ -2,68 +2,63 @@
-- UTs testing calicoctl, STs testing single host and multihost systems
-  using calicoctl and calico/node to create Calico networked containers.  These
-  tests run within the `calico/test` Docker image.
-  for the UT and ST codebase.  Also see the Makefile for
-  details on how the STs (`make st`) and UTs (`make ut`) are run using the
-  `calico/test` image.
-- Release scripts used to validate our documentation and to cut a new branch
-  for a specific release.
-
-It pulls together a number of related repositories that provide subsets of
-function for running Calico in a containerized environment.  These related
-respositories are listed below.
-
-## Calico Libraries
-
- - [calico](https://github.com/projectcalico/calico): This repo contains the
-   implementation of Felix which is the heart of Calico networking.  Felix
-   interfaces with the Linux kernel to configure routes and ACLs
-   that control network policy connectivity. In `calico/node`, Felix runs as a
-   separate process.  It is installed directly into the `calico/node` image.
-
- - [libcalico](https://github.com/projectcalico/libcalico): Contains a variety
-   of helper methods and classes for manipulating the Calico data in an etcd
-   datastore, IP address management and useful namespace utilities to
-   manipulating container interfaces.  The libcalico library is used by
-   `calicoctl`.  It is imported as a standard Python library into the
-   `calicoctl` Python codebase.  libcalico also contains the `calico/test` Docker
-   image used for running both UTs and STs testing Calico with Docker.
-
-## Calico Orchestrator Integration Plugins
+The following information details which artifacts are built from which
+repositories.
+
+## Core Components
+
+### [Felix](https://github.com/projectcalico/felix)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| `felix` | Binary | Felix interfaces with the Linux kernel to configure routes and ACLs that control network policy and connectivity. |
+
+
+### [Calico-Containers](https://github.com/projectcalico/calico-containers)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| `calico/node`| Docker Image |  The Docker image used to run Felix and the BIRD BGP agent (for distributing routes between hosts).  See [`calico_node` directory]({{site.baseurl}}/{{page.version}}/reference/architecture/components) for details. |
+| `calicoctl`| Binary | The command line tool for interacting with the Calico API.  See [`calicoctl` directory]({{site.baseurl}}/{{page.version}}/reference/calicoctl) for implementation of calicoctl. |
+
+## Libraries
+
+### [libcalico](https://github.com/projectcalico/libcalico)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| libcalico | pypi library | Contains a variety of helper methods and classes for manipulating the Calico data in an etcd datastore, IP address management and useful namespace utilities to manipulating container interfaces. |
```

Comment:
You've got inconsistent use of "`" for the artifacts.

Personally I'd leave them off entirely.